src/openeo_odc/map_to_odc.py

```diff
@@ -21,15 +21,12 @@ def map_to_odc(graph, odc_env, odc_url):
         if cur_node.parent_process: #parent process can be eiter reduce_dimension or apply
             if cur_node.parent_process.process_id == 'reduce_dimension':
                 kwargs['dimension'] = cur_node.parent_process.content['arguments']['dimension']
-        if set(cur_node.arguments.keys()) in [{'x', 'y'}, {'x', }, {'data', 'value'}, {'base', 'p'}]:
-            nodes[cur_node.id] = map_required(cur_node.id, cur_node.content, kwargs)
-        elif 'data' in tuple(cur_node.arguments.keys()):
-            nodes[cur_node.id] = map_required(cur_node.id, cur_node.content, kwargs)
-        elif 'min' in tuple(cur_node.arguments.keys()):
-            nodes[cur_node.id] = map_required(cur_node.id, cur_node.content, kwargs)
-        elif 'id' in tuple(cur_node.arguments.keys()):
+        param_sets = [{'x', 'y'}, {'x', }, {'data', 'value'}, {'base', 'p'}, {'data', }]
+        if 'id' in tuple(cur_node.arguments.keys()):
             # This should be only load_collection
             nodes[cur_node.id] = map_load_collection(cur_node.id, cur_node.content)
+        elif (params in set(cur_node.arguments.keys()) for params in param_sets):
+            nodes[cur_node.id] = map_general(cur_node.id, cur_node.content, kwargs)
```

Comment:
Is the `min` parameter not needed anymore?

Reply:
Before this change we looked if the `param_sets` were exactly equal to the `parameters` used in the processes. Now we only look if the `param_sets` are in the `parameters`, but there can be additional parameters... so as the `id` parameter is used in `load_collection`, but `load_collection` also has an `x` in its parameter list, it would also be `True` for the `elif params in ...` statement, which is why I put the `if id ... ` first. an other solution would be, that we first have a look at the `cur_node.name` and if the name is `load_collection`, we use `map_load_collection`. of course this should than also be the first if, because for the `load_collection` the `elif params in ... ` is also `True`.

Reply:
As we now only check if `x` is in the `parameters` and not if `x` is equal to the `parameters`, we do not need the `min` anymore, because it is used in the `clip` process with `x` and with `max`. :)

Reply:
> Is load_collection the only process with a parameter id? 

No, there's also load_result with an id.

Reply:
Thx @m-mohr for the input!

Maybe in this case for `load_collection` and `load_result` checking the process name is actually the best option, because those two processes just need to be handled differently than most other ones. 

So, @ValentinaHutter please just check the processes name for `load_collection` (this could then actually be the first check because it is very specific). And I assume that we'll need to do the same for `load_result`, just calling a different - not yet implemented - mapping function.

Reply:
The `load_collection` process is now handled properly. For `load_result` I opened an issue. It will be addressed in a separate PR.